translations/de/hints/bryukh.html

```diff
@@ -0,0 +1,40 @@
+<body data-author="bryukh">
+<div class="question-answer">
+    <div class="question">
+        Ich habe keine Idee, wie ich anfangen soll, diese Mission zu lösen
+    </div>
+    <div class="answer">
+        Versuch nicht das Rad neu zu erfinden und benutze die <a href="http://docs.python.org/3/library/functions.html#int">int() Funktion</a>.
+        Python hat schon ein Tool um eine Zahl in verschiedene Basen zu konvertieren.
+<pre class="brush: python">
+int("1A", 16) == 170
+int("101", 2) == 5
+    </pre>
+    </div>
+</div>
+<div class="question-answer">
+    <div class="question">
+        Ich benötige etwas Hilfe, um mit der Mission weiterfahren zu können
+    </div>
+    <div class="answer">
+        Aufgepasst bei "falschen" Fällen, weil "int()" wird einen Fehler (ValueError) dafür ausgeben.
+        Lies dazu, wie Du <a href="http://docs.python.org/3/tutorial/errors.html#exceptions">Ausnahmen handhaben</a> kannst.
+    (Schaue bei <a href="http://docs.python.org/3/library/exceptions.html?highlight=valueerror#ValueError">ValueError</a>).
+    </div>
+</div>
+<div class="question-answer">
+    <div class="question">
+        Ich bin auf halbem Wege verloren. Ich brauche Hilfe!
+    </div>
+    <div class="answer">
+        Anstelle für jeden Fall einzeln im Voraus zu prüfen, kannst Du einfach die Fehlerbehandlung einbinden und bist für diese Fehler gerüstet.
+<pre class="brush: python">
+try:
+    do_someting # etwas ausgeben
+except ValueError: # except (Ausnahmehandabung) ohne Typenangabe ist schlechte Praktik
```

Comment:
> `(Ausnahmehandabung)`

- `Ausnahmebehandlung` is the correct word
- The parentheses are not necessary.